bee-network/src/service/host.rs

```diff
@@ -312,8 +323,8 @@ async fn process_command(command: Command, senders: &Senders, peerlist: &PeerLis
     Ok(())
 }
 
-async fn process_internal_event(int_event: InternalEvent, senders: &Senders, peerlist: &PeerList) -> Result<(), Error> {
-    match int_event {
+async fn process_internal_event(i_event: InternalEvent, senders: &Senders, peerlist: &PeerList) -> Result<(), Error> {
+    match i_event {
```

Comment:
Not a big fan of these kind of names tbh

Reply:
I want to transaction to `icommand` and `ievent` to still indicate that it's an internal command/event, but reduces the noise significantly.

Reply:
You see noise everywhere! Common `internal_event` is not noisy at all, if two words are noisy we have bigger problems!

Reply:
haha, fair enough!